net/redisclient.go

```diff
@@ -359,7 +359,9 @@ func (r *RedisRingClient) Close() {
 	r.once.Do(func() {
 		r.closed = true
 		close(r.quit)
-		r.ring.Close()
+		if r.ring != nil {
```

Comment:
Ah re-introduced :)

Reply:
Yeah, I decided to plug nil check as quick fix but we should update ratelimit initialization such that it does not create redis client with nil ring